requirements.txt

```diff
@@ -1,20 +1,19 @@
-arrow==0.8.0
-biopython==1.68
-boto==2.46.1
-bz2file==0.98
-ConfigArgParse==0.11.0
-gensim==0.13.2
-Logbook==1.0.0
-numpy==1.16
-pep8==1.7.0
-pluggy==0.4.0
-py==1.4.33
-pytest==3.0.7
-python-dateutil==2.6.0
-requests==2.20.0
-scipy==0.19.0
-six==1.10.0
-smart-open==1.5.1
-tox==2.7.0
-tox-pyenv==1.0.3
-virtualenv==15.1.0
+arrow
```

Comment:
please pin the versions. This will install the latest versions, which may break the code.

Reply:
This was one of the issues when moving up to Python 3.7. Several of the older versions of the libraries were incompatible with Python 3.7, so installing the latest versions would definitely reduce incompatibilities.